babel.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = (api) => {
   api.assertVersion('^7.4.4');
 
   const presets = [
-    '@babel/preset-env',
+    ['@babel/preset-env', { useBuiltIns: 'entry', corejs: { version: 3.6, proposals: true } }],
```

Comment:
what's this for?

Reply:
This modify the core-js polyfills to only pull in the polyfills required for our browserlist configuration. See [core-js documentation](https://github.com/zloirock/core-js#babelpreset-env) for more details.